control_node/control_node/nodes/control_node.py

```diff
@@ -343,13 +274,19 @@ def __init__(self):
         )
         main_sm.add_state("BURN", Burn(), transitions={"runner_removed": "ACQ"})
 
-        self.cam_to_laser = None
         self.curr_track = None
         self.laser_client = LaserNodeClient(self)
         self.camera_client = CameraNodeClient(self)
         self.runner_tracker = Tracker()
+
         blackboard = Blackboard()
         blackboard.main_node = self
+        blackboard.calibration = Calibration(
```

Comment:
I'm down to have things tied to the board, or to the node, let me know if there is any distinction you'd like to keep to.

Reply:
I'm almost no preference, but having everything on the node seems like more levels of indirection in each state machine state (blackboard -> node -> what I need). I probably should have just passed it to the constructor of whatever states need it so that the dependencies of each state are clear.

Reply:
Passing it in to the init sounds good long term. I can go back and make that the paradigm for all of the nodes and move away from really using the blackboard at all.

Reply:
I'm actually doing that now :)

Reply:
Awesome, thanks